integration/tests/cook/test_basic.py

```diff
@@ -2473,3 +2474,30 @@ def test_data_local_support(self):
         self.assertEqual(201, resp.status_code, resp.text)
         job = util.load_job(self.cook_url, uuid)
         self.assertEqual(True, job['data_local'], job)
+
+    @unittest.skipIf(os.getenv('COOK_TEST_SKIP_RECONCILE') is not None,
+                     'Requires not setting the COOK_TEST_SKIP_RECONCILE environment variable')
+    def test_reconciliation(self):
+        """
+        This test relies on 4 running jobs being seeded in Datomic *before* Cook Scheduler starts
+        up, so that when it does start up and the reconciler runs, it can reconcile those tasks
+        (i.e. discover that they are not actually running in Mesos and mark them as failed). See
+        scheduler/datomic/data/seed_running_jobs.clj for the details of this job-seeding.
+        """
+        info = util.scheduler_info(self.cook_url)
+        cook_start_dt = dateutil.parser.parse(info['start-time'])
+        cook_start_ms = (cook_start_dt - datetime.fromtimestamp(0, tz=utc)).total_seconds() * 1000
+        start = int(cook_start_ms - 240000)
+        end = util.current_milli_time()
+        resp = util.jobs(self.cook_url, user='seed_running_jobs_user',
```

Comment:
Instead of querying for the jobs, what if we put the UUIDs in an env. variable (which we could use to trigger running the test.) I doubt this would be flaky, but it lets you avoid the weird timestamp math.

Reply:
Yeah, I'm not super happy with the timestamp math, but I think the environment variable approach would make the workflow when running locally be not ideal. You'd have to have the environment variable defined both when you seed the jobs and when you're running the test (which might be from an IDE).